src/blockchain/chain.rs

```diff
@@ -348,6 +419,15 @@ impl Chain {
         }
     }
 
+    /// This asynchronous function checks if the payment deadline associated with the most recent sell block
```

Comment:
I think we don't need to document if a function is async, or not, since that's part of the API anyway